.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why do we need these settings here in addition to `package.json`? eslint supports both locations.